Tetris/Tetris/StartScreen.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  StartScreen.swift
+//  Tetris
+//
+//  Created by Megha Sharma on 02/06/22.
+//
+
+import SwiftUI
+
+struct GrowingButton: ButtonStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .padding()
+            .background(Color(red: 0, green: 0, blue: 0.5))
+            .foregroundColor(.white)
+            .clipShape(Capsule())
+            .scaleEffect(configuration.isPressed ? 1.2 : 1)
+            .animation(.easeOut(duration: 0.2), value: configuration.isPressed)
+    }
+}
+
+struct StartScreen: View {
+    @State var userName = "USER1234"
+    @State private var animateGradient = true
+    
+    var body: some View {
+        ZStack{
+            LinearGradient(colors: [.purple, .yellow], startPoint: .topLeading, endPoint: .bottomTrailing)
+                .ignoresSafeArea()
+                .onAppear {
+                        withAnimation(.linear(duration: 2.0).repeatForever(autoreverses: true)) {
+                            animateGradient.toggle()
+                        }
+                }
+            VStack{
+                Text("High Score").font(.headline).padding()
+                Text("1080").font(.largeTitle).padding()
```

Comment:
Move this to variable.
In real view this will be a computed value

Reply:
Like username

Reply:
@sharadJay Can we do same for colors like primary and secondary so that if it will change in future we can edit just primary or secondary values?